Plugins/GitUIPluginInterfaces/ObjectId.cs

```diff
@@ -480,7 +480,7 @@ public override string ToString()
         /// <exception cref="ArgumentOutOfRangeException"><paramref name="length"/> is less than zero, or more than 40.</exception>
         [Pure]
         [NotNull]
-        public unsafe string ToShortString(int length = 10)
+        public unsafe string ToShortString(int length = 8)
```

Comment:
Are there any usages left that specify a length? If we want to make this consistent across the board, perhaps remove the parameter altogether.

Reply:
There are no (ObjectId external) usages of the parameter now.
I do not want to remove it as it is part of the plugin interface.
Also, I expect that a unique length may be needed in some situations like reports. That may use Git though for the abbr length, but this opens for the possibility to add safer lengths (longer than 10 chars).

Reply:
> Are there any usages left that specify a length?

Yes, `CommitIdColumnProvider`.